### PR TITLE
Db for Etcd changes

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: block-storage-attacher/tests/e2e/conf/developer.conf [hunk 1]
```diff
@@ -50,11 +50,13 @@ HELM_CHART_REPO=$HELM_CHART_REPO
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
 PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
-ETCD_SET1=etcd:https://portal-ssl495-47.bmix-dal-yp-b15b4c9e-cc62-4a8b-900d-895f414f2528.1152061296.composedb.com:18308
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/developer.conf]: Please change the `IMAGE_REGISTRY` in this file.

File: block-storage-attacher/tests/e2e/e2e-tests/portworx_secret.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: px-ibm
-  namespace: portworx
+  name: px-etcd-certs
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/e2e-tests/portworx_secret.yaml]: should this be `px-ibm` and namespace be `portworx`?

File: block-storage-attacher/tests/e2e/conf/developer.conf (hunk 1 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/conf/developer.conf]: This uses staging registry I Should change to staging

File: block-storage-attacher/tests/e2e/e2e-tests/portworx_secret.yaml (hunk 2 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/e2e-tests/portworx_secret.yaml]: No those values are right

**Pull request merged**